### PR TITLE
Merge branch on advance care plans to main branch

### PR DESCRIPTION
**Pull request by stutibagri**

### CODE REVIEW

File: analysis/os_reports/dataset_definition_os_reports.py
Comment (sophie-julian) [inline analysis/os_reports/dataset_definition_os_reports.py:1]: start date now updated to march 2019

File: project.yaml
Comment (sophie-julian) [inline project.yaml:1]: the current yaml just uses wild card for each quality measures, so place and cause of death are within one wildcard

File: analysis/os_reports/04_eol_WP2_quality_indicators.R [hunk 1]
```diff
@@ -1,249 +1,52 @@
...
-      legend.spacing.y = unit(0.1, "cm")
-    )
-}
-
-
 # Code settings -----------------------------------------------------------
 # need to decide on this
-startdate <- dmy("01-06-2019")
-enddate <- dmy("30-06-2023")
+startdate <- dmy("01-12-2018")
```
Comment (sophie-julian) [inline analysis/os_reports/04_eol_WP2_quality_indicators.R:22]: start date has been changed to march 2019. quarters have now been updated with 18 full quarters from march 2019 onwards (so first quarter has been removed).

File: analysis/os_reports/04_eol_WP2_quality_indicators.R [hunk 2]
```diff
@@ -256,261 +59,454 @@ df <- read_csv(file = here::here("output", "os_reports", "input_os_reports.csv.g
...
+
+#Average number of care plan codes recorded per person during the last 90 days (3 months) of life by place of death
+
+#Quarterly breakdown 
+
+#Rounded estimates
+
+acp3m_pod_quarters_rounded <- df %>%
+  group_by(study_quarter, pod_ons_new) %>%
+  summarise(count = sum(has_careplan == 'TRUE', na.rm = TRUE),
```
Comment (sophie-julian) [inline analysis/os_reports/04_eol_WP2_quality_indicators.R:202]: does this mean the mean number of times ACP is coded for in the group of people who have ACP recorded? would we not want the mean to include people who didn't have an ACP recorded as well?

**Review by sophie-julian — COMMENTED** [review]
have a few comments, also have been some updates to the code - new start date and quarters. I think we need to think about what the duration outputs will look like. I'm also thinking because of all the conflicts perhaps it's easier to put this analysis on a new separate R file e.g. 04_etc etc then it should make resolving the conflicts much easier

File: analysis/os_reports/04_eol_WP2_quality_indicators.R (hunk 2 above)
Comment (stutibagri) [reply analysis/os_reports/04_eol_WP2_quality_indicators.R:202]: Hmmm my logic was that we have a sense of what proportion have ACP recorded anyway, and if we included those who had no ACP in this measure, we wouldn't have a correct sense of how many records of advance care planning appear for those with at least one mention of it

**Pull request merged**